doc/jcli/vote.md

```diff
@@ -39,4 +77,54 @@ jcli transaction seal --staging vote-tally.staging
 jcli transaction auth --staging vote-tally.staging --key committee.sk
 jcli transaction to-message --staging vote-tally.staging > vote-tally.fragment
 jcli rest v0 message post --file vote-tally.fragment
-```
\ No newline at end of file
+```
+
+### Private
+To tally private votes, all committee members are needed.
```

Comment:
Apart from creating the tally we need to document the creation of a vote plan.

Reply:
Another note on that: not "all committee members" but the number of them specified in the threshold.

Reply:
Actually is all committee members for now, since the threshold is not still implemented.